assets/src/ba_data/python/ba/_gameactivity.py

```diff
@@ -959,7 +959,7 @@ def spawn_player_spaz(
         # Move to the stand position and add a flash of light.
         spaz.handlemessage(
             StandMessage(
-                position, angle if angle is not None else random.uniform(0, 360)
+                position, angle or random.uniform(0, 360)
```

Comment:
In case `angle` is 0 that won't work as expected.
```py
>>> 0 or 1337
1337
```

Reply:
But 'Optional or Default' syntax looks good in general. Note that it simply evaluates to `x if bool(x) else y` https://stackoverflow.com/a/3914686/17444908